src/main.js

```diff
@@ -144,7 +144,10 @@ function configureCommandlineSwitchesSync(cliArgs) {
 		'disable-color-correct-rendering',
 
 		// override for the color profile to use
-		'force-color-profile'
+		'force-color-profile',
+
+		// Enable experimental font access for terminal ligatures
+		`enable-features='FontAccess'`
```

Comment:
Did this work ? I would expect appendSwitch to ignore this, it should instead be

`app.commandLine.appendSwitch('enable-features', 'FontAccess');`

Reply:
Yep it worked, can change it though

Reply:
lets keep it separate for consistency.